webui/sg-view/nuxt.config.js

```diff
@@ -57,6 +57,6 @@ module.exports = {
   ** Allow overriding from environment
   */
   env: {
-    HISTORY_URL: process.env.HISTORY_URL || 'http://localhost:8001/history'
+    HISTORY_URL: process.env.HISTORY_URL
```

Comment:
I think if we are not going to set a default then we can get rid of this.

Reply:
Ok ... wasn't sure if it was used elsewhere or wanted for some other reason. It's removed.